### ISSUE TITLE
Presentation mode shows previous and next slide

### ISSUE THREAD

**Issue by Irubataru** [pwmt/zathura#686]
If I make a presentation in beamer and the aspect ratio of the presentation isn't the same as the monitor it is displayed on, then presentation mode in zathura shows the slide (page) before and after at the top and bottom. I'd rather that it showed black bars. This would also make sure it was centered when showing the first and last slide.

Probably related to #103 

How it looks now:
![image](https://github.com/user-attachments/assets/96422774-ac19-4089-9a1d-fa9038d216ea)

How I expect it to look:
![image](https://github.com/user-attachments/assets/bee6f571-e029-4b27-99b9-97a314f351af)

Zathura version:
```
zathura 0.5.8
girara 0.4.4 (runtime: 0.4.4)
(plugin) ps (0.2.8) (/usr/lib/zathura/libps.so)
(plugin) pdf-mupdf (0.4.4) (/usr/lib/zathura/libpdf-mupdf.so)
(plugin) djvu (0.2.10) (/usr/lib/zathura/libdjvu.so)
```

**Comment by tobsto** [comment]
I would like to second that. Such a feature would be very much appreciated. It is for example the standard way in evince.

**Comment by kuon** [comment]
Yes being able to use zathura for presentations would be very appreciable.

Basically only one page should be displayed at a time, and page-up/down should move to previous/next page without moving the page frame.

**Comment by ehula** [comment]
This feature is desperately needed.
